controllers/members.ts

```diff
@@ -19,6 +19,21 @@ interface GetParams {
   query?: string
 }
 
+export const getMemberByMemberCode = async (memberCode: string) => {
+  const { data } = await client.query<{ users: User[] }>({
+    query: USER_BY_MEMBER_CODE,
+    variables: {
+      memberCode,
+    },
+  })
+
+  if (data.users.length) {
+    return { user: data.users[0], error: null }
+  }
+
+  return { error: 'Usuario no encontrado', user: null }
```

Comment:
```suggestion
  return { error: 'Usuario no encontrado' }
```

I think the other way is more semantic, first check if there is an error, is called "early return"